toonz/sources/image/tiio.cpp

```diff
@@ -159,6 +162,33 @@ void initImageIo(bool lightVersion) {
   TFileType::declare("rgb", TFileType::RASTER_IMAGE);
   Tiio::defineWriterProperties("rgb", new Tiio::SgiWriterProperties());
 
+  // ffmpeg
+  if (Ffmpeg::checkFfmpeg()) {
```

Comment:
Please add a following line before the line 166:
`#if (defined(x64))`
and after the line 189, please add `#endif`

In Ffmpeg::checkFfmpeg() you are checking the value in Preferences, and Preferences tries to see TOONZPROFILES path from registry (in its ctor). This behavior seems to abort t32bitsrv.exe since registry paths are different between 32bit and 64bit binaries.
Adding `#if (defined(x64))` is to avoid this problem by skipping Ffmpeg::checkFfmpeg() on running t32bitsrv.exe. This check must be only needed for OpenToonz_1.0.exe.